src/main/webapp/recommendation-map.html

```diff
@@ -59,11 +60,11 @@ <h3>Want to view a recommendation?</h3>
       <!-- View Recommendation Area -->
       <div id="rec-container">
         <p id="category-header">- Places to Visit -</p>
-        <h3>Sky Garden</h3>
+        <h3 id="place-title">Sky Garden</h3>
         <h4>Location / Address</h4>
         <p id="rec-address">51.510881° N, -0.083751° W</p>
         <h4>Description</h4>
-        <p>Sky Garden is a great place to visit, with really good
+        <p id="place-recommendation">Sky Garden is a great place to visit, with really good
           views of London. You can enjoy the 360 degree view
           of the city - for free! They are public gardens and 
           definitely worth a visit when you have the time.
```

Comment:
Shall we remove the default values from here? Given that we are populating them for each recommendation already

Reply:
Good idea, thanks